src/pages/JoinComplete/JoinComplete.KakaoChatroom.tsx

```
@@ -1,5 +1,6 @@
-import { Check, Copy } from "lucide-react";
+import { Copy } from "lucide-react";
 import { forwardRef, useState } from "react";
+import toast from "react-hot-toast";
```

Biome (2.1.2)

[error] This import is unused.

Unused imports might be the result of an incomplete refactoring.
Unsafe fix: Remove the unused imports.


(lint/correctness/noUnusedImports)